Src/Base/AMReX_Extension.H

```diff
@@ -48,7 +48,7 @@
 #elif defined(__HIP_DEVICE_COMPILE__)
 #define AMREX_PRAGMA_SIMD 
 
-//#elif defined(_OPENMP) && (_OPENMP >= 201307) && !defined(__PGI)
+//#elif defined(AMREX_USE_OMP) && (AMREX_USE_OMP >= 201307) && !defined(__PGI)
```

Comment:
```suggestion
//#elif defined(AMREX_USE_OMP) && defined(_OPENMP) && (_OPENMP >= 201307) && !defined(__PGI)
```
Although this is commented out.